jupyter-org-client.el

```diff
@@ -416,6 +416,357 @@ the `syntax-table' will be set to that of the REPL buffers."
   (jupyter-org-with-src-block-client
    (cl-call-next-method)))
 
+;;; Org helper functions
+
+;;;###autoload
+(defun jupyter-org-insert-src-block (&optional below)
+  "Insert a src block above the current point.
+With prefix arg BELOW, insert it below the current point.
+
+If point is in a block, copy the header to the new block"
+  (interactive "P")
+  (if (org-in-src-block-p)
+      (let* ((src (org-element-context))
+             (start (org-element-property :begin src))
+             (end (org-element-property :end src))
+             (lang (org-element-property :language src))
+             (switches (org-element-property :switches src))
+             (parameters (org-element-property :parameters src))
+             location)
+        (if below
+            (progn
+              (goto-char start)
+              (setq location (org-babel-where-is-src-block-result nil nil))
+              (if (not  location)
+                  (goto-char end)
+                (goto-char location)
+                (goto-char (org-element-property :end (org-element-context))))
+              (insert (format "#+BEGIN_SRC %s\n\n#+END_SRC\n\n"
+                              (concat lang (and lang " ")
+                                      switches (and switches " ")
+                                      parameters (and parameters " "))))
+              (forward-line -3))
+          ;; after current block
+          (goto-char (org-element-property :begin (org-element-context)))
+          (insert (format "#+BEGIN_SRC %s\n\n#+END_SRC\n\n"
+                          (concat lang (and lang " ")
+                                  switches (and switches " ")
+                                  parameters (and parameters " "))))
+          (forward-line -3)))
+
+    ;; Not in a src block, just insert a block
+    (beginning-of-line)
+    (insert (format "#+BEGIN_SRC %s\n\n#+END_SRC\n" (completing-read "Language: " (mapcar 'car org-babel-load-languages))))
```

Comment:
Again, I would like to use `org-element-interpret-data`, see https://github.com/dzop/emacs-jupyter/pull/45/files#r261337038